packages/cf-component-card/src/CardBlock.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  padding: '1rem 1.4rem 1.26667rem',
+  borderTop: `1px solid ${theme.colorGrayLight}`,
```

Comment:
@koddsson I can say a lot of things about our card control patterns but I don't have the full picture of what's needed here so let me know if I/design team can be helpful in some way. :)

Some questions I'd like to ask anyway:
- Are the grey lines part of the pattern?
- Are there existing places in the UI where we have a block of text above a control?